content/id/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md

```diff
@@ -0,0 +1,449 @@
+    name: php-apache
+  minReplicas: 1
+  maxReplicas: 10
+  metrics:
+  - type: Resource
+    resource:
+      name: cpu
+      target:
+        type: Utilization
+        averageUtilization: 50
+status:
+  observedGeneration: 1
+  lastScaleTime: <some-time>
+  currentReplicas: 1
+  desiredReplicas: 1
+  currentMetrics:
+  - type: Resource
+    resource:
+      name: cpu
+      current:
+        averageUtilization: 0
+        averageValue: 0
+```
+
+Perhatikan bahwa kolom `targetCPUUtilizationPercentage` telah diganti dengan array yang disebut `metrics`.
+Metrik penggunaan CPU adalah *resource metric*, merepresentasikan sebagai persentase sumber daya
+ditentukan pada wadah pod. Perhatikan bahwa kamu dapat menentukan metrik sumber daya selain CPU. Secara default,
+satu-satunya metrik sumber daya lain yang didukung adalah memori. Sumber daya ini tidak mengubah nama dari klaster
+ke klaster, dan harus selalu tersedia, selama API `metrics.k8s.io` tersedia.
+
+Kamu juga dapat menentukan metrik sumber daya dalam hal angka langsung, bukan sebagai persentase dari
+nilai yang diminta, dengan menggunakan `target.type` dari `AverageValue` sebagai ganti `Utilization`, dan
+menyetel _field_ `target.averageValue` yang sesuai, bukan `target.averageUtilization`.
+
+Ada dua jenis metrik lainnya, keduanya dianggap sebagai *metrik khusus*: metrik pod dan
+metrik objek. Metrik ini memungkinkan untuk memiliki nama yang spesifik untuk klaster, dan membutuhkan lebih banyak
+pengaturan pemantauan klaster lanjutan.
+
+Jenis metrik alternatif yang pertama adalah *metrik pod*. Metrik ini mendeskripsikan pod, dan
+dirata-ratakan bersama di seluruh pod dan dibandingkan dengan nilai target untuk menentukan jumlah replika.
+Mereka bekerja seperti metrik sumber daya, kecuali bahwa mereka *hanya* mendukung jenis `target` dari `AverageValue`.
+
+Metrik Pod ditentukan menggunakan blok metrik seperti ini:
+
+```yaml
+type: Pods
+pods:
+  metric:
+    name: packets-per-second
+  target:
+    type: AverageValue
+    averageValue: 1k
+```
+
+Jenis metrik alternatif kedua adalah *metrik objek*. Metrik ini mendeskripsikan perbedaan
+objek di namespace yang sama, bukan mendeskripsikan pod. Metriknya belum tentu
+diambil dari objek; mereka hanya mendeskripsikannya. Metrik objek mendukung jenis `target`
+baik `Value` dan `AverageValue`. Dengan `Value`, target dibandingkan langsung dengan yang dikembalikan
+metrik dari API. Dengan `AverageValue`, nilai yang dikembalikan dari API metrik khusus dibagi
+dengan jumlah pod sebelum dibandingkan dengan target. Contoh berikut adalah YAML
+representasi dari metrik `requests-per-second`.
+
+```yaml
+type: Object
+object:
+  metric:
+    name: requests-per-second
+  describedObject:
+    apiVersion: networking.k8s.io/v1beta1
+    kind: Ingress
+    name: main-route
+  target:
+    type: Value
+    value: 2k
+```
+
+Jika Anda memberikan beberapa blok metrik seperti itu, HorizontalPodAutoscaler akan mempertimbangkan setiap metrik secara bergantian.
```

Comment:
Use "kamu" instead of "anda"
```suggestion
Jika kamu memberikan beberapa blok metrik seperti itu, HorizontalPodAutoscaler akan mempertimbangkan setiap metrik secara bergantian.
```